cli/actions/__tests__/dev.test.js

```diff
@@ -40,7 +40,9 @@ describe('dev', () => {
     clientURL: mockURL,
     serverURL: mockURL,
     reactHotLoader: false,
-  });
+  },
+  {},
+  []);
```

Comment:
Can we default these upstream so they don't need to be set if they're not needed?